torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -458,13 +462,20 @@ def render(  # type: ignore[override]
         x_zp = None
         w_scale = None
         w_zp = None
+        is_int8_binary_with_bias = False
```

Comment:
We should support epilogue fusion for arbitrary args. Why do we handle "binary" fusion in a special way here?

Reply:
Thanks for the comment and remove this flag.